src/node_http_parser.cc

```diff
@@ -187,7 +188,58 @@ struct StringPtr {
   size_t size_;
 };
 
+class Parser;
+
+struct ParserComparer {
+  bool operator()(const Parser* lhs, const Parser* rhs) const;
+};
+
+class ConnectionsList : public BaseObject {
+ public:
+    static void New(const FunctionCallbackInfo<Value>& args);
+
+    static void All(const FunctionCallbackInfo<Value>& args);
+
+    static void Idle(const FunctionCallbackInfo<Value>& args);
+
+    static void Active(const FunctionCallbackInfo<Value>& args);
+
+    static void Expired(const FunctionCallbackInfo<Value>& args);
+
+    void Push(Parser* parser) {
+      allConnections.insert(parser);
+    }
+
+    void Pop(Parser* parser) {
+      allConnections.erase(parser);
+    }
+
+    void PushActive(Parser* parser) {
+      activeConnections.insert(parser);
+    }
+
+    void PopActive(Parser* parser) {
+      activeConnections.erase(parser);
+    }
+
+    SET_NO_MEMORY_INFO()
+    SET_MEMORY_INFO_NAME(ConnectionsList)
+    SET_SELF_SIZE(ConnectionsList)
+
+ private:
+    std::set<Parser*, ParserComparer> allConnections;
+    std::set<Parser*, ParserComparer> activeConnections;
```

Comment:
nit:
```suggestion
    std::set<Parser*, ParserComparer> all_connections_;
    std::set<Parser*, ParserComparer> active_connections_;
```
we use snake case with a `_` at the end for member variables

Reply:
👍